python-avd/pyavd/schema/utils.py

```diff
@@ -0,0 +1,3 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
```

Comment:
Why this file is added? is it by mistake?

Reply:
Good point. Yes it was a mistake on my end.